src/codegen.cpp

```diff
@@ -519,14 +518,12 @@ typedef struct {
     bool vaStack;      // varargs stack-allocated
     bool sret;
     int nReqArgs;
-    std::vector<bool> boundsCheck;
-    std::vector<bool> inbounds;
 
     CallInst *ptlsStates;
     Value *signalPage;
 
-    llvm::DIBuilder *dbuilder;
     bool debug_enabled;
+    bool is_inbounds{false};
```

Comment:
..... Does changing `typedef struct { ... } jl_codectx_t;` to `struct jl_codectx_t { ... };` help?